js/editable-js.js

```diff
@@ -61,7 +57,13 @@
         var output = document.querySelector('#output code');
 
         output.classList.add('fade-in');
-        executeInteractiveExample(exampleCode);
+
+        try {
+          executeInteractiveExample(exampleCode);
```

Comment:
As this function no consist of just one line, I reckon we can get rid of it and just do `new Function(exampleCode)();` right inside the `try` block.